src/couch_replicator/src/couch_replicator_api_wrap.erl

```diff
@@ -256,6 +270,13 @@ open_doc_revs(#httpdb{} = HttpDb, Id, Revs, Options, Fun, Acc) ->
             ),
             #httpdb{retries = Retries, wait = Wait0} = HttpDb,
             Wait = 2 * erlang:min(Wait0 * 2, ?MAX_WAIT),
+            ?LOG_NOTICE(#{
+                what => retry_request,
+                in => replicator,
+                url => Url,
+                delay => Wait / 1000,
```

Comment:
needs unit suffix, one of:

- `_ms` for msec
- `_sec` for sec
- `_hr` for hours
- `_us` for microseconds